src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -15,6 +15,7 @@
 public class AddressBook implements ReadOnlyAddressBook {
 
     private final UniquePersonList persons;
+    //private final MeetingsList meetings;
```

Comment:
Also this

Reply:
The address book doesn't need meetings right? I think it can be removed.